### DISCUSSION TITLE
searchConfig: assemblies vs assemblyNames?

### DISCUSSION THREAD

**Discussion by bbimber** [GMOD/jbrowse-components#2286] [General] [answered]
Hello,

I'm wrapping my head around text search config. If this is an accurate example:

https://github.com/GMOD/jbrowse-components/blob/a8de8f144e260e210e098f280d8dc458dd3a6fc0/products/jbrowse-cli/test/data/config.json

Why use the term assemblyNames on the track and "assemblies" on search config?

**Comment by cmdcolin** [comment, marked as answer]
I went ahead and changed it from "assemblies"->"assemblyNames" for consistency. thanks for making a note of this. it is worthwhile consistency to have. they were technically just separate parts of the config so weren't synced up.
